src/content/getting-started/cloud-provider-accounts/vmware-cloud-director/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+linkTitle: Preparing VMware Cloud Director for CAPI
+title: Preparing VMware Cloud Director for Cluster API
+description: How to set up your VMware Cloud Director Cluster tenant to run Giant Swarm management clusters and workload clusters under your jurisdiction.
+weight: 80
+user_questions:
+  - What are the VMware Cloud Director prerequisites for use with Giant Swarm?
+  - What do I need to configure in VMware Cloud Director in order to run Giant Swarm clusters?
+  - What user permissions do I need to run Giant Swarm clusters in VMware Cloud Director?
+menu:
+  main:
+    identifier: gettingstarted-infraprovider-clouddirector
+    parent: gettingstarted-infraprovider
+last_review_date: 2023-10-03
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+---
+In order to run the Giant Swarm platform in your VMware Cloud Director (VCD) environment, a number of prerequisites must be satisfied to support Cluster API Provider VMware Cloud Director (CAPVCD).
+
+## Cloud Director infrastructure
+
+VMware Cloud Director must run with API version 36 compatibility at least. However, it is highly recommended to run VCD 10.4 or above as it includes shared virtual service IPs.
```

Comment:
These two numbers have different meanings. From the [docs](https://developer.vmware.com/apis/1170/), I see that API version v36.0 means product version v10.3. Can we mention this here? Note how their documentation uses `v` and dot notation, so I suggest that here.

```suggestion
VMware Cloud Director must run at least with API version v36.0 compatibility (minimum version VCD v10.3). It is highly recommended to run VCD v10.4 or above as it includes shared virtual service IPs.
```

Reply:
Thanks